tokenExchange/src/app/validation.js

```diff
@@ -29,7 +29,7 @@ async function jwtValidator(jwtToken) {
         }
         if (checkAudience(aud) !== -1) {
             if (checkIssuer(issuer) !== -1) {
-                if (organization === undefined || checkRoles(role) !== -1) {
+                if (organization === undefined || checkRoles(role) !== -1 ||  role.startsWith('pg-')) {
```

Comment:
Questo perchè?

Reply:
Dopo aver controllato se l'utente non ha il campo organization(PF) o il role è tra quelli permessi(PA), controllo che role inizi con 'pg-' per non lanciare l'errore "Role not allowed"